conjur/controller/policy_controller.py

```diff
@@ -35,5 +35,5 @@ def load(self):
         except requests.exceptions.HTTPError as http_error:
             if http_error.response.status_code == 422:
```

Comment:
If the error is from different type 404? 
I think we need to add this fix in more general place, no?

Reply:
i think this is a policy related error handling. So this is the place. agree server should not return 404 on policy not found. weird no?